src/index.ts

```diff
@@ -174,6 +172,8 @@ function setVenvWrapper(hre: HardhatRuntimeEnvironment, venvPath: string) {
 
 // add venv wrapper or docker wrapper of starknet
 extendEnvironment((hre) => {
+    hre.starknetJs = new StarknetLegacyWrapper(hre.config.starknet.networkConfig);
```

Comment:
Is the new `starknetJs` property of `hre` meant to be used both internally and by users?